CONTRIBUTING.md

```diff
@@ -180,22 +180,23 @@ As mentioned before, tests are split into groups thanks to [jest-runner-groups](
  */
 ```
 
-**Run**
+##### Run
 
-To run unit tests you can either use
+To run unit tests, you can either use:
 
 * npm task `lerna-test:unit` (`npm run lerna-test:unit`) in root folder to run them all
```

Comment:
I think we can actually remove this from the doc: `lerna-test` that we have now will run the `test` script in the `package.json` of each package ([example](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/tracing/package.json#L14)) with in turn is an alias for the unit group.

Reply:
We could keep it for consistency (because the packages also contain both `test` and `test:unit` that are in fact the same thing). Otherwise, it might be misleading - why can I run both `npm run test:unit` and `npm run test` in the package but only `npm run lerna-test` in the root?

Reply:
Ok, makes sense. My reasoning was simply to avoid confusion and leave only one but I see where you're coming from. I've already approved the PR, I'll leave the conversation open so that the next reviewer can chime in if they want.